pages/index.tsx

```diff
@@ -1,4 +1,6 @@
 import HomePage from '@/views/home'
+import { Suspense } from 'react'
+import BalloonLoader from '@/components/loader'
```

Comment:
Neither of these imports are being used in here. I dont even wanna know what Suspense is